### ISSUE TITLE
Pokemon Preprocessing error 

### ISSUE THREAD

**Issue by zsz00** [LiorSinai/DenoisingDiffusion.jl#3]
Downsample images

```
Progress:  34%|██████████████▏                          |  ETA: 0:00:14
AssertionError: top < bottom && left < right

Stacktrace:
 [1] get_bbox(img::Matrix{RGB{N0f8}}, thresh::Float64)
   @ Main ~/codes/julia/DenoisingDiffusion.jl/examples/Pokemon_preprocessing.ipynb:17
 [2] get_bbox(img::Matrix{RGB{N0f8}})
   @ Main ~/codes/julia/DenoisingDiffusion.jl/examples/Pokemon_preprocessing.ipynb:2
 [3] macro expansion
   @ ~/codes/julia/DenoisingDiffusion.jl/examples/Pokemon_preprocessing.ipynb:3 [inlined]
 [4] top-level scope
   @ ~/.julia/packages/ProgressMeter/sN2xr/src/ProgressMeter.jl:938
```

**Comment by LiorSinai** [comment]
This is a data issue. The assertion will be raised if something is improper. Is this case, it is failing because the Pkm_img2561.jpg is blank and it cannot draw a bounding box.

However this error should be handled correctly, so I'll push an update shortly.

**Comment by LiorSinai** [comment]
Solved with commit 13fcf3763659b39b7ba4b7ba5aa0f97dfbaea140
